inst/@sym/intersect.m

```diff
@@ -98,7 +98,7 @@
 %! % empty input
 %! A = sym([1 2]);
 %! C = intersect(A, []);
-%! assert (isequal (C, sym([])))
+%! assert (isequal (size(C), [1, 0]))
```

Comment:
Would `assert (isempty (C))` work?

Reply:
(advantage is test would pass on both new and old sympy)

Reply:
> Would `assert (isempty (C))` work?

Yes, this too is likely to work.